typescript/package.json

```diff
@@ -17,7 +17,7 @@
     "componentize": "bunx jco componentize world.js --wit wit -o ts-typescript.wasm -d all"
   },
   "devDependencies": {
-    "@bytecodealliance/jco": "1.8.1",
+    "@bytecodealliance/jco": "file:../../jco",
```

Comment:
There is not a release yet that includes this fix:

- https://github.com/bytecodealliance/ComponentizeJS/pull/168